neuraxle/checkpoints.py

```diff
@@ -35,6 +35,17 @@ def __init__(self, force_checkpoint_name: str = None):
         BaseStep.__init__(self)
         self.force_checkpoint_name = force_checkpoint_name
 
+    def setup(self, step_path: str, setup_arguments: dict):
+        self.set_checkpoint_path(step_path)
+
+    def handle_transform(self, data_container: DataContainer) -> Any:
+        self.save_checkpoint(data_container)
```

Comment:
Should the action of saving a checkpoint return a data_container? (e.g.: if the saver renames the hash for it to be fine as a filename). I don't know, what would you do?